forge-gui/res/cardsfolder/upcoming/wild_magic_surge.txt

```diff
@@ -0,0 +1,8 @@
+Name:Wild Magic Surge
+ManaCost:R R
+Types:Instant
+A:SP$ Destroy | ValidTgts$ Permanent.OppCtrl | RememberLKI$ True | TgtPrompt$ Select target permanent an opponent controls | SubAbility$ DBDigUntil | SpellDescription$ Destroy target permanent an opponent controls. Its controller reveals cards from the top of their library until they reveal a permanent card that shares a card type with that permanent. They put that card onto the battlefield and the rest on the bottom of their library in a random order.
```

Comment:
Previous effects like that only worked if it got destroyed, which is why it would only remember then

But this one should also Dig if target regenerates etc.

You may want to add `AlwaysRemember` support in DestroyEffect